### ISSUE TITLE
[BUG] [Formatter]When html tag is written in django variable tag, line break is introduced and variable tag is destroyed.

### ISSUE THREAD

**Issue by Taikono-Himazin** [djlint/djLint#735]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows 10
 - Python Version : 3.9.13
 - djLint Version : v2023.8.7(VSCode Extension Version)
 - template language: django_template

## Issue
```html
<p>{{ some_val | default:"some string 1<br />some string 2" }}</p>
``` 
should not be changed, or
```html
<p>
     {{ some_val | default:"some string 1<br />some string 2" }}
</p>
```
I'd like you to It actually looks like this:
```html
<p>
     {{ some_val | default:"some string 1
     <br />
     some string 2" }}
</p>
```
This causes a line break in response to the <br> between default"", and it is not recognized as a variable tag by django.

## How To Reproduce
Add `<p>{{ some_val | default:"some string 1<br />some string 2" }}</p>` in vscode and run djlint.

Thank you.

**Comment by DostalJ** [comment]
New to the djlint, but I guess it is caused by `format_js=true` config.

> Format contents of script tags using js-beautify. See [js-beautify](https://github.com/beautify-web/js-beautify/blob/main/python/jsbeautifier/javascript/options.py) for all configuration options. **Template syntax is not [fully supported](https://github.com/beautify-web/js-beautify/issues) in supported.**

**Comment by oliverhaas** [comment]
Was fixed somewhere between 1.32.0 and 1.33.0, so this can be closed.
